action/recipes/edk2.go

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+
+// Package recipes / edk2
+package recipes
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"dagger.io/dagger"
+	"github.com/9elements/firmware-action/action/container"
+)
+
+// edk2 builds edk2
+func edk2(ctx context.Context, client *dagger.Client, common *commonOpts, dockerfileDirectoryPath string, opts *edk2Opts, artifacts *[]container.Artifacts) error {
+	// TODO: get blobs in place!
+	envVars := map[string]string{
+		"WORKSPACE":      common.containerWorkDir,
+		"EDK_TOOLS_PATH": "/tools/Edk2/BaseTools",
+	}
+
+	// Spin up container
+	containerOpts := container.SetupOpts{
+		ContainerURL:      common.sdkVersion,
+		MountContainerDir: common.containerWorkDir,
+		MountHostDir:      common.repoPath,
+		WorkdirContainer:  common.containerWorkDir,
+	}
+	myContainer, err := container.Setup(ctx, client, &containerOpts, dockerfileDirectoryPath)
+	if err != nil {
+		return err
+	}
+
+	// Copy over the defconfig file
+	// TODO ??? Is there even config to copy over?
```

Comment:
I think you already answered yourself in the action.yml file stating that this file only contains the build args, no?

Reply:
Yeah, I think so.